### PR TITLE
moved repeatTimes to a const in benchmark tests.

### PR DESCRIPTION
**Pull request by khnom5000**
also removed _ from ranges

### CODE REVIEW

File: itertools_benchmark_test.go [hunk 1]
```diff
@@ -23,7 +24,7 @@ func BenchmarkIter(b *testing.B) {
 }
 
 func BenchmarkRepeatEdge(b *testing.B) {
-	stringLength, repeatTimes := 100000, 100000000
+	stringLength := 100000
```
Comment (h-dav) [inline itertools_benchmark_test.go:27]: Please can you make this 100 and a constant just like `repeatTimes`

**Review by h-dav — CHANGES_REQUESTED** [review]
Just one change :)

**Review by h-dav — APPROVED** [review]
Lovely stuff

**Comment by khnom5000** [comment]
thx

**Pull request merged**